test_data/yam/whole_disk_and_boot_unattended.yaml

```diff
@@ -0,0 +1,22 @@
+---
+disks:
+  - name: vda
+    table_type: xfs
+    partitions:
+      # whole disk formatted
+      - name: vda
+        fstype: xfs
+        mountpoints: [/home]
+        type: disk
+  - name: vdb
+    table_type: gpt
+    partitions:
+      - name: vdb1
+        fstype: btrfs
+        mountpoints: ["/var", "/usr/local", "/srv", "/root", "/opt", "/"]
+  - name: vdc
+    table_type: gpt
+    partitions:
+      - name: vdc1
+        fstype: vfat
+        mountpoints: ["/boot/efi"]
```

Comment:
are we checking any mountpoints in the verifications for those two modules or the fstype?
I didn't find it, maybe we should not add them for any disk, because for example this efi mountpoint doesn't make sense in x86_64 non-uefi, right?

Reply:
> are we checking any mountpoints in the verifications for those two modules or the fstype? I didn't find it, maybe we should not add them for any disk, because for example this efi mountpoint doesn't make sense in x86_64 n`on-uefi, right?

It's in [validate_lsblk](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/63cf62b7fe533130d65263df645533cc6aeaac4a/lib/filesystem_utils.pm#L495), and I removed the uefi mountpoint for your suggestion. Thanks.